images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -18,6 +18,12 @@ function Get-BazeliskVersion {
     return "Bazelisk $bazeliskVersion"
 }
 
+function Get-BicepVersion {
+    ($(bicep --version) | Out-String) -match  "bicep version (?<version>\d+\.\d+\.\d+)" | Out-Null
```

Comment:
nitpick: `(bicep --version | Out-String)` should work too

Reply:
Much cleaner. Updated.